.gitignore

```
@@ -1,4 +1,4 @@
-lambda-registrator/lambda-registrator
+modules/lambda-registrator/lambda-registrator
```

There isn't a file named `modules/lambda-registrator/lambda/registrator` and the `lambda-registrator/` dir does not build any executable artifacts. I think this line should be removed.

There is an executable file named `consul-lambda-registrator/lambda-registrator` that I think should be deleted.. I think it must have got checked in by accident at some point. Maybe that is what the original line was referring to? cc @erichaberkorn

---

Yah, I'm pretty sure this is there to prevent the executable from being checked in 👍

---

OK, so I think we should delete the file: `consul-lambda-registrator/lambda-registrator` and make the following change to `.gitignore`:

```suggestion
consul-lambda-registrator/lambda-registrator
```

---

Addressed in [this](https://github.com/hashicorp/terraform-aws-consul-lambda-registrator/pull/31/commits/6c3002485879866884f042d824e038930c1ca1cd) push.